docs/tutorial/tutorial.md

```diff
@@ -200,6 +226,10 @@ class Board extends React.Component {
       squares: Array(9).fill(null),
     };
   }
+  render() {
+    // TODO
```

Comment:
This is not what I see when I follow along from the [starter CodePen](https://codepen.io/gaearon/pen/JNYBEZ?editors=0010). Can we either make it similar or make it clear the code is just being skipped?